mxpy-up.py

```diff
@@ -131,6 +131,40 @@ def migrate_old_elrondsdk() -> None:
     else:
         logger.info(f"Old config path does not exist: {old_config_path}.")
 
+    # Remove vmtools, if exists. Has to be re-downloaded, and re-compiled (libwasmer-related issues will arise otherwise).
```

Comment:
This PR also contains the merge from main to `feat/next` - it's OK for now, since it wasn't a big change, but in the future we should avoid that, when possible.